vite-plugin-css-name-normalizer.ts

```diff
@@ -39,7 +39,8 @@ export default async function cssNameNormalizer(): Promise<Plugin> {
         })
         .forEach((key) => {
           const entry = bundle[key];
-          const fileName = kebabCase(key.split('.').shift());
+          const filePath = kebabCase(key.split('.').shift());
+          const fileName = filePath.split('/').pop();
```

Comment:
This could probably be a one-liner but it looked hideous when I did it. Open to suggestions

Reply:
This might warrant a unit test at a future date

Reply:
Good call